draft-ietf-ppm-dap.md

```diff
@@ -1443,10 +1469,10 @@ round trip is initiated by the leader.
 
 #### Leader Continuation {#aggregation-leader-continuation}
 
-The leader begins each round of continuation for a report share based on its
-locally computed prepare message and the previous PrepareStep from the helper.
-If PrepareStep is of type `failed`, then the leader acts based on the value of
-the `ReportShareError`:
+The Leader begins each round of continuation for a report based on its locally
+computed prepare message and the previous PrepareStep from the helper. If
```

Comment:
Hmm, doesn't the Leader need the previous PrepareStep in order to compute its next prep share?